etc/orthography.tsv

```diff
@@ -0,0 +1,88 @@
+īh$	ī χ	shīh;hut-līh;hut-līh;sam-tlīh
+ōh$	ō ʔ	tin-ā-ōh;tlōh;kōh;mōh;mōh
+a	a	wai-oot;swaika;soi-kan;kah-tin;ikalla
+ā	ā	pu-kwā-num;pu-kwā´-num;pook-wā-num;stil-tāmsh;shā-wut
+ă	ă	sĭttăk;sātsă-tăkum;sātsă-tăkum;kslă-koo-num;kslă-koo-num
+ts	ts	tsa-tsa-ta-kum;sātsă-tăkum;tloo-tsi-tsi;tloo-tsi-tsi;tats-guika
+e	e	kewlh;ske-ki-thlil;akag-eluk;tā-gool-taine-tīn;tse-ke
+ē	ē	swē-ka;sim-ēm;to-kwunē-e;tē-nā-se;kloh-tēlt
+ĕ	ĕ	tsalĕ-gedit;a-ko-ko-ut´-tĕ-na;in-kĕ-shēl´-ni-kuk;snĕt;sumĕ-hai-it-tshin
+ee	e e	keet;keet;keet;skee-ool-ki
+x	x	nux-soo;nux-soo;nux-soo;nux-soo;tsim-aux-shin
+χ	χ	ki-χi;kitl-χow
+x$	x	spsis-sax;ko-kwinix;tla-la-hlam´-ix;moax;moax
+h	h	i-hling-a;ē-hlin;kah-tin;sahlt;pahtl-ke
+^h	h	hun-nāh;hā-tlel-a;hā-puh;ha-push;ha-push
+h$	h	youh;in-tlah;skul-tum-ih;hā-puh;til-koolh-im-un-nah
+i	i	wai-oot;i-hling-a;ē-hlin;enika;swaika
+ī	ī	wīns;tā-gool-taine-tīn;kī-ka;tshīl;tshīl
+ĭ	ĭ	sĭttăk;smoo-ītlĭts;mā-kĭn;kuai-ĭgoot;toa-whĭ-ai
+j	j	n-jada;ti-koot-jung-a;ti-koot-jung-a;di-jasha;ti-jas-ka
+g	g	i-thlung-a;i-hling-a;akag-eluk;tā-gool-taine-tīn;n-gunna
+k	k	pook-wā-num;enika;swaika;soi-kan;swē-ka
+gh	gʰ	ti-ow-gha;ti-tla-ghă;uh-hi-ghutti;uh-hi-ghutti;shā-wut-uh-ghutti
+gs	g s	yungs-a-kīn
+k$	k	sĭttăk;akag-eluk;’kl-wām-ilk;walak;kl-ku-am-ilk
+kh	kʰ	’kha;skh-ai-oos;ka-kloo-khluh´-ā;uh-a-kha-wo;’khai
+ks	k s	salks-māna;salks-māna;sa-ykso;tahks-ta;muksn
+kw	kʷ	pu-kwā-num;pu-kwā´-num;to-kwunē-e;a-poo-kwunse;ani-kwul
+kwh	kʷʰ	ut-la-kwhl
+kws	kʷ s	is-kws-sum-ēm´;hakws-smuk-wătshi
+l	l	i-thlung-a;i-hling-a;ē-hlin;stil-tāmsh;im-ilkh
+m	m	im-ilkh;skul-tum-ih;tloo-tshimun;smoo-tlats;sim-ēm
+ms	m s	timst;timst
+mh$	m h	et-shumh;kwalimh
+n	n	i-thlung-a;i-hling-a;pu-kwā-num;pu-kwā´-num;pook-wā-num
+nh	ɲ	tsh-kwonh-ama;kla-nhl-kwun-ilh
+ns	ns	insua-nimān;insua-nimān;a-poo-kwunse;insua-en-tang;a-poo-kwunse
+b	b	be-ba-da;ā-bo;ābo;in-tsă-bum;tubo-osh
+p	p	a-pa;hā-puh;ha-push;ha-push;kush-aoomp-kins
+bb	bː	muh-muh-gubba
+bs	b s	obsin
+ps	p s	inl-tst-tso-ōps´;spsis-sax;kul-ōpst;an-o-is-kumps´-e;ka-tlas-kumps´-e
+ph$	p h	āh-alph
+r	r	nun-tse-ra;skow-rē;ni-ta-rā;nin-ta-ra;nin-tshur-rā
+s	s	itl-na-huit-soo;swe-wil-is-sonil;swe-wil-is-sonil;ta-na-asi;tl-kasts
+z	z	ka-kutz-ko;χa-da-hit-zoo;na-ā-tzoo-tzoo;na-ā-tzoo-tzoo;skat-soo-za
+d	d	jā-da;χa-da;n-jada;jā-da-hut-so;χa-da-hit-zoo
+t	t	stōbsh;stil-tāmsh;stil-tāmsh;kah-tin;in-tlah
+dh	dʰ	sai-a-dha;tzia-dho;tei-dha;stā-dhe;kwā-dha
+ds	d s	kwa-kwa-dsanai;tin-ne-adsoh;skads-oot-la;kah-kup-adsi
+th	tʰ	i-thlung-a;ske-ki-thlil;tha-thus;na-ilth-lash;na-ilth-lash
+tt	tː	sĭttăk;uh-hi-ghutti;uh-hi-ghutti;shā-wut-uh-ghutti;shā-wut-uh-ghutti
+o	o	youh;soi-kan;tshuoi;swe-wil-is-sonil;now-te-nun´
+ō	ō	kō-wus;stōbsh;tōāl-o;is-kōi´;nis-tōlis
+ŏ	ŏ	wŏa-puh;nukŏăt;tŏ-tōa;tŏ-tōa
+oo	uː	wai-oot;pook-wā-num;tloo-tshimun;tā-gool-taine-tīn;smoo-tlats
+w	w	pook-wā-num;kō-wus;shā-wut;kewlh;’kl-wām-ilk
+u	u	youh;i-thlung-a;pu-kwā-num;pu-kwā-num;pu-kwā´-num
+y	y	til-koolh-im-yoot;tai-yuz;nai-yoo;nai-yoo;n-yai
+ū	ū	tūll
+tl	tɬ	
+’k	ʔk	
```

Comment:
These should be pre-glottalized, not double articlulation.

Reply:
Just to be sure (first profile and all), this means `ʔ k` instead of `ʔk`?

Reply:
It depends if it is a pre-glottalized ( ˀk ) or a distinct segment ( ʔ k ). Most of the time it is a matter of theoretical decision of the author.

What I usually do, unless otherwise stated by the author or available from some other documentation, is to manually inspect the cognates: if there is something that likely corresponds to a glottal stop before the plosive (most of the time, another plosive), I keep them as separate segments; if not, I mark it as pre-glottalized. But if you have descriptions in the inventory of at least one language, you should try to follow it (hopefully for Haida at least there is material).

Reply:
This ( http://www.phonology.or.kr/xe/?module=file&act=procFileDownload&file_srl=840&sid=19cbdb6c1d4a8d549db7ed1291bc40cf ) suggests, following Maddieson, that they mostly pre-glottalized sounds indeed (thus, `ˀk`).

Reply:
Thank you, that helps! Yes, having a look at Haida (and also Sahaptin) might help. I'll do that.